nixos/tests/forked-daapd.nix

```diff
@@ -0,0 +1,44 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  configDir = "/var/lib/foobar";
+  port = 3689;
+in {
+  name = "forked-daapd";
+  meta = with pkgs.stdenv.lib; {
+    maintainers = with maintainers; [ flyfloh ];
+  };
+
+  nodes = {
+    forked_daapd =
+      { pkgs, ... }:
+      {
+        services.forked-daapd = {
+          enable = true;
+          libraryPaths = [ /media/music ];
```

Comment:
`/media/music` should not be imported into the store. I guess this should be `"/media/music"`.

Reply:
Fixed